### DISCUSSION TITLE
Claimed rewards from self pool not showing up in wallet transactions

### DISCUSSION THREAD

**Discussion by TAvant** [Chia-Network/chia-blockchain#13689] [Support]
I had 3.5 XCH in my self pool that I claimed and it has been several days and still have not seen the transaction come through in my wallet. I am running Ubuntu 20.04.5 LTS and Chia Blockchain 1.6.1-dev0.

**Comment by shaneo257** [comment]
its a known issue, your wallet balance will increase tho

**Reply by TAvant** [reply]
Its been since October 15th and I still haven't seen the the transaction come through. Do I need to update my Chia Blockchain to see this transaction go through?
